packages/headless/src/features/advanced-search-queries/advanced-search-queries-state.ts

```diff
@@ -13,6 +13,11 @@ export interface AdvancedSearchQueriesDefaultFiltersState {
    * The initial lq filter, or large query expression filter
    */
   lq: string;
+
+  /**
+   * The initial dq filter, or disjunction query expression filter
```

Comment:
I like putting the equation like the others so users understand what it's for
`((q AND aq) OR dq) AND cq)`